robottelo/config/validators.py

```diff
@@ -42,6 +42,12 @@
             default=NetworkType.IPV4.value,
         ),
         Validator('server.is_ipv6', is_type_of=bool, must_exist=False),
+        Validator(
+            'server.default_os_search_query',
+            is_type_of=str,
+            must_exist=False,
```

Comment:
This means that the parameter is **forbidden** from existing and the run will fail in any case (because if it isn't in the config file, we also provide default value):
```
dynaconf.validator.ValidationError: server.default_os_search_query cannot exists in env main
```